plugins/modules/route53.py

```diff
@@ -122,7 +122,7 @@
     type: str
   wait:
     description:
-      - Wait until the changes have been replicated to all Amazon Route 53 DNS servers.
+      - Wait until the changes have been replicated to all Amazon Route53 DNS servers.
```

Comment:
Amazon seems to call it "Route 53", not "Route53" (https://en.wikipedia.org/wiki/Amazon_Route_53).